main.go

```diff
@@ -32,30 +33,44 @@ var (
 	blue   = color.New(color.FgBlue).SprintFunc()
 )
 
-func main() {
-	config := flag.String("c", "", "config path")
-	name := flag.String("n", "", "name of specific test to run")
-
-	var envs pkg.KeyValuePairs
-	flag.Var(&envs, "e", "environment variable")
+func init() {
```

Comment:
This doesn't really do much :P  Maybe make it log.SetFlags with the std and short file guff?

Reply:
my reasoning is, given a command line tool:
- line number output is only valuable to the developer, not the user
- timestamps are recorded by the terminal, the user doesn't need them

This call is to ensure logs are going to `stderr`, but do not contain any prefixes.

Thoughts?

Reply:
Oh of course, I forgot it was a tool for a second, yep.  This makes total sense.